README.md

```diff
@@ -8,12 +8,24 @@ Work in progress... Partial documentation ahead.
 ### Quick start
 
 To run the server with minimal configuration export the `KUBECONFIG` environment variable and run:
-``` 
+```
 go run ./cmd/service-account-issuer-discovery/main.go --hostname=<issuer-of-cluster>
 ```
 Or pass the `kubeconfig` as a flag:
-``` 
+```
 go run ./cmd/service-account-issuer-discovery/main.go --kubeconfig=<path-to-my-kubeconfig> --hostname=<issuer-of-cluster>
 ```
 
 Retrieve the `well-known` document by querying `/.well-known/openid-configuration`.
+
+### Helm Charts
+
+service-account-issuer-discovery
+
+* europe-docker.pkg.dev/gardener-project/releases/charts/gardener/service-account-issuer-discovery:VERSION
+
+### Container Images
+
+service-account-issuer-discovery
+
+* europe-docker.pkg.dev/gardener-project/releases/gardener/service-account-issuer-discovery:VERSION
```

Comment:
I think it is better just mention that the details of the available docker images and helm charts can be found in the releases page and add a link to https://github.com/gardener/service-account-issuer-discovery/releases/. 

This way if we for some reason update the OCI repository, we do not have to update the README.md file.

Reply:
How about now? 

The text is using the word "container" instead of "Docker" since the latter is a proprietary term and it's use is discouraged in general context.